src/styles/components/_product-card.scss

```diff
@@ -0,0 +1,105 @@
+.bsds-product-card
+{
+
+  a[href]::after,
+  a[href]::before
+  {
+    content: "";
+    display: block;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+  }
```

Comment:
What does this do? Do we still need it?

Reply:
This is the pseudo link element to expand hitbox to the image.